inji-web/src/utils/api.ts

```diff
@@ -13,7 +13,8 @@ export enum MethodType {
 
 export class api {
 
-    static mimotoHost = window._env_.MIMOTO_HOST;
+    static mimotoHost = "http://localhost:3010";
+    // static mimotoHost = window._env_.MIMOTO_HOST;
```

Comment:
uncomment this and comment the localhost.

Reply:
yes uncommented that